bundles/c/c_lexer.moon

```diff
@@ -33,14 +33,7 @@ howl.aux.lpeg_lexer ->
     span '/*', '*/'
   }
 
-  char_constant = P"'" * any({
-    "\\" * any {
-      R('07') * R('07') * R('07'),
-      'x' * xdigit * xdigit,
-      S'abfnrtv\\\'"?'
-    },
-    1
-  }) * "'"
+  char_constant = span("'", "'", '\\')
```

Comment:
Could you add an example of the multi-character literal syntax to `bundles/c/misc/example.c`?